browser.js

```diff
@@ -0,0 +1,23 @@
+(function(scope){
+    "use strict";
+
+    function base64ToBrowser (buffer) {
+        return window.btoa([].slice.call(new Uint8Array(buffer)).map(function(bin) { return String.fromCharCode(bin) }).join(""));
+    }
+
+    function imageToBase64Browser (urlOrImage, param) {
+        if (!("fetch" in window && "Promise" in window)) {
+            return Promise.reject("[*] It's image2base64 not compatible with your browser.");
+        }
+        return fetch(urlOrImage, param || {}).then(function(response){
+            return response.arrayBuffer()
+        }).then(base64ToBrowser);
+    }
+
+    if (typeof module !== "undefined") {
```

Comment:
I have a question, even using react we need to validate `module.exports`?

Reply:
I suppose that the validation isn't required it but I'm not sure if it another libraries may need it, so if you think it's safe we can remove it

Reply:
Let's awaiting, for now, later we go remove, right?